.env.example

```diff
@@ -71,4 +71,5 @@ GROUP_HOSTNAMES=whoami.im.c.127.0.0.1.nip.io
 ADMIN_USER_EMAIL=andreas@dhis2.org
 ADMIN_USER_PASSWORD=somepassword
 
-S3_BUCKET=test-db-manager-bucket
+DEFAULT_USER_EMAIL=im-default@dhis2.org
```

Comment:
I don't like having a generic user like this and default doesn't really reveal it's purpose. Let's call it what it is... im-e2e-test@dhis2.org or such.